charts/adaptive/values.yaml

```diff
@@ -443,6 +443,58 @@ tensorboard:
     storageClass: ""
   emptyDir:
     sizeLimit: null
+  
+mlflow:
+  # If set to true, jobs will use MLflow for experiment tracking
+  # A separate MLflow tracking server will be deployed
+  # Takes priority over tensorboard if both are enabled
+  enabled: false
+  imageUri: python:3.12-slim
```

Comment:
let's use the official image: `ghcr.io/mlflow/mlflow:v3.1.1` to avoid a pip install every time the container is launched